ui/tasks.py

```diff
@@ -30,3 +34,42 @@ def post_video_to_edx(video_id):
         (endpoint.full_api_url, getattr(resp, "status_code", None))
         for endpoint, resp in response_dict.items()
     ]
+
+
+@app.task
+def batch_update_video_on_edx(video_keys, chunk_size=1000):
+    """
```

Comment:
Might be good to add unit tests for these too.  Here is an example to get started, since celery unit tests can be tricky:


```python
def test_batch_update_video_on_edx(mocker):
    """
    batch_update_video_on_edx should call batch_update_video_on_edx_chunked for each chunk of video keys
    """
    mock_batch = mocker.patch("ui.tasks.batch_update_video_on_edx_chunked")
    group_mock = mocker.patch("ui.tasks.celery.group", autospec=True)
    all_ids = list(range(1, 101))
    batch_update_video_on_edx(all_ids, chunk_size=10)

    assert group_mock.call_count == 1
    for i in list(range(10)):
        mock_batch.assert_any_call(all_ids[i*10:i*10+10])
```